test-configs.yaml

```diff
@@ -749,6 +749,11 @@ device_types:
             - 'defconfig+CONFIG_RAMDOMIZE_BASE=y'
           kernel: ['v3.']
 
+  fsl-ls2088a-rdb:
```

Comment:
Please follow the alphabetical order of the devices.

Reply:
oops, it's alphabetical for NXP.  ;)   Will fix.